### PR TITLE
Update Ukrainian language translations

### PR DESCRIPTION
**Pull request by Vadimkin**
This pull request adds Ukrainian translations for previously missing keys in the localization file. I also changed informal pronoun "твоєї" to formal "вашої" for better consistency and formality.

### CODE REVIEW

File: ghost/i18n/locales/uk/comments.json [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
-    "Sign in": "",
-    "Sign up now": "",
-    "Start the conversation": "",
-    "This comment has been hidden.": "",
-    "This comment has been removed.": "",
-    "Upgrade now": "",
-    "Yesterday": "",
-    "You want to report this comment?": "",
-    "Your request will be sent to the owner of this site.": ""
+    "{{amount}} characters left": "Залишилось {{amount}} символів",
```
Comment (akabuda050) [inline ghost/i18n/locales/uk/comments.json:2]:
```suggestion
    "{{amount}} characters left": "Залишилось {{amount}} символ(ів)",
```

File: ghost/i18n/locales/uk/comments.json [hunk 2]
```diff
@@ -1,73 +1,73 @@
...
+    "{{amount}} comments": "{{amount}} коментарів",
```
Comment (akabuda050) [inline ghost/i18n/locales/uk/comments.json:3]:
```suggestion
    "{{amount}} comments": "{{amount}} коментар(ів)",
```

Comment (akabuda050) [reply ghost/i18n/locales/uk/comments.json:3]: і т.д.

**Review by akabuda050 — CHANGES_REQUESTED** [review]

File: ghost/i18n/locales/uk/comments.json (hunk 1 above)
Comment (akabuda050) [reply ghost/i18n/locales/uk/comments.json:2]: хммм. можливо не треба й заборочуватися. бо варіанти такі як `Залишилось(вся) 1/2  символ(ів)`  буде занадто :)

**Review by akabuda050 — APPROVED** [review]

**Comment by ErisDS** [comment]
Hey @Vadimkin, thanks for your contributions 🙏  I believe the changes here have been covered by https://github.com/TryGhost/Ghost/pull/20709, but if I've missed anything please feel free to open a new changeset ☺️

**Pull request closed without merging**